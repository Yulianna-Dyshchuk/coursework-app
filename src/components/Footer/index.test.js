import React from "react";
import { shallow } from "enzyme";
import { Footer } from "./index";

describe("Footer component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
