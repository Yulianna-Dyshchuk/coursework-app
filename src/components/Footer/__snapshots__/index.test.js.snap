// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly without props 1`] = `
<footer>
  <div>
    <FooterList
      key="0"
      list={
        Array [
          Object {
            "label": "Home",
            "to": "/",
          },
          Object {
            "label": "About",
            "to": "/about",
          },
          Object {
            "label": "Prices",
            "to": "/prices",
          },
          Object {
            "label": "Contacts",
            "to": "/contacts",
          },
          Object {
            "label": "Portfolio",
            "to": "/portfolio",
          },
        ]
      }
      title="Pages"
    />
    <FooterList
      key="1"
      list={
        Array [
          Object {
            "label": "Home",
            "to": "/",
          },
          Object {
            "label": "About",
            "to": "/about",
          },
          Object {
            "label": "Prices",
            "to": "/prices",
          },
        ]
      }
      title="Information"
    />
    <FooterList
      key="2"
      list={
        Array [
          Object {
            "label": "Phone: +380950189548",
            "to": "tel:+380950189548",
            "type": "tel",
          },
          Object {
            "label": "Gmail: example@example.com",
            "to": "mailto:example@example.com",
            "type": "mailto",
          },
          Object {
            "label": "Instagram: <custom-link>",
            "target": "_blank",
            "to": "https://www.instagram.com/",
          },
          Object {
            "label": "Facebook: <custom-link>",
            "target": "_blank",
            "to": "https://www.facebook.com/",
          },
        ]
      }
      title="Contacts"
    >
      <div>
        <FooterSocialLink
          icon={
            <Memo
              className=" "
            />
          }
          key="0"
          to=""
        />
        <FooterSocialLink
          icon={
            <Memo
              className=" "
            />
          }
          key="1"
          to=""
        />
        <FooterSocialLink
          icon={
            <Memo
              className=" "
            />
          }
          key="2"
          to=""
        />
        <FooterSocialLink
          icon={
            <Memo
              className=" "
            />
          }
          key="3"
          to=""
        />
      </div>
    </FooterList>
  </div>
  <div>
    <WithStyles(ForwardRef(Typography))
      variant="body2"
    >
      @All rights reserved! 2021
    </WithStyles(ForwardRef(Typography))>
  </div>
</footer>
`;
